local.properties

```diff
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Wed Feb 26 13:48:45 EST 2020
-sdk.dir=/home/nate/Android/Sdk
+#Sun Mar 08 14:37:01 EDT 2020
+sdk.dir=C\:\\Users\\admin\\AppData\\Local\\Android\\Sdk
```

Comment:
android studio automatically updated the sdk location. If you merge this pull request make sure this gets changed back to your local path

Reply:
I think their .gitignore is out of whack... I see several files in this commit that should be filtered out by .gitignore.